data/json/itemgroups/activities_hobbies.json

```
@@ -168,6 +168,25 @@
       { "item": "60l_methanol_tank", "prob": 1 }
     ]
   },
+  {
+    "type": "item_group",
+    "id": "nice_chess",
```

the id for this group is not descriptive, and not even used correctly time to time, being either `nice_chess` or `cool_chess`
```suggestion
    "id": "loose_chess_pieces",
```